Dockerfile.thirdeye

```diff
@@ -0,0 +1,77 @@
+#
```

Comment:
why a different path? can we stick to the same path as in the parent repo?
https://github.com/apache/incubator-pinot/blob/master/docker/images/pinot-thirdeye/Dockerfile

Reply:
Look at the original dockerfile. You will know. We need to import from the root path. Docker cannot mount beyond its current path. Alternative is what they have done with git clone and thats clumsy

Reply:
@asldevi look at the PR description above

Reply:
👍